pkg/registry/core/pod/storage/eviction.go

```diff
@@ -246,7 +246,7 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		refresh := false
 		err = retry.RetryOnConflict(EvictionsRetry, func() error {
 			if refresh {
-				pdb, err = r.podDisruptionBudgetClient.PodDisruptionBudgets(pod.Namespace).Get(context.TODO(), pdbName, metav1.GetOptions{})
+				pdb, err = r.podDisruptionBudgetClient.PodDisruptionBudgets(pod.Namespace).Get(ctx, pdbName, metav1.GetOptions{})
```

Comment:
just glancing over here and not looking deep. Can you help understand if the error handling below will be affected by this? Specifically the place where `err == wait.ErrWaitTimeout` - with the context I assume the context may be cancelled. Any chance it will be ever recognized as a timeout and mistakenly reported that there were conflicts?